src/openrct2/management/Award.cpp

```diff
@@ -263,21 +263,15 @@ static bool award_is_deserved_best_staff(int32_t activeAwardTypes)
     if (activeAwardTypes & EnumToFlag(ParkAward::MostUntidy))
         return false;
 
-    auto peepCount = 0;
     auto staffCount = 0;
     auto staffTypeFlags = 0;
-    for (auto peep : EntityList<Peep>(EntityListId::Peep))
+    for (auto staff : EntityList<Staff>(EntityListId::Peep))
     {
-        if (peep->AssignedPeepType == PeepType::Staff)
-        {
-            staffCount++;
-            staffTypeFlags |= (1 << static_cast<uint8_t>(peep->AssignedStaffType));
-        }
-        else
-        {
-            peepCount++;
-        }
+        staffCount++;
+        staffTypeFlags |= (1 << static_cast<uint8_t>(staff->AssignedStaffType));
```

Comment:
```suggestion
        staffTypeFlags |= (1 << EnumValue(staff->AssignedStaffType));
```

Reply:
Actually can simplify this further and remove that variable.